docs/how-xtrabackup-works.md

```diff
@@ -1,123 +1,149 @@
-    INSTANCE FOR BACKUP` or `LOCK TABLES FOR BACKUP`.
-
-xtrabackup tries to avoid backup locks and `FLUSH TABLES WITH READ LOCK`
-when the instance contains only InnoDB tables. In this case, xtrabackup
-obtains binary log coordinates from `performance_schema.log_status`. `FLUSH
-TABLES WITH READ LOCK` is still required in MySQL 8.0 when xtrabackup is
-started with the `--slave-info`. The `log_status` table in Percona
-Server for MySQL 8.0 is extended to include the relay log coordinates, so no locks are
-needed even with the `--slave-info` option.
-
-!!! admonition "See also"
-
-    [MySQL Documentation: LOCK INSTANCE FOR BACKUP](https://dev.mysql.com/doc/refman/8.0/en/lock-instance-for-backup.html)
-
-When backup locks are supported by the server, xtrabackup first copies
-InnoDB data, runs the `LOCK TABLES FOR BACKUP` and then copies the MyISAM
-tables. Once this is done, the backup of the files will
-begin. It will backup .frm, .MRG, .MYD, .MYI, .CSM,
-.CSV, `.sdi` and `.par` files.
-
-After that xtrabackup will use `LOCK BINLOG FOR BACKUP` to block all
-operations that might change either binary log position or
-`Exec_Master_Log_Pos` or `Exec_Gtid_Set` (i.e. source binary log coordinates
-corresponding to the current SQL thread state on a replication replica) as
-reported by `SHOW MASTER/SLAVE STATUS`. xtrabackup will then finish copying
-the REDO log files and fetch the binary log coordinates. After this is completed
-xtrabackup will unlock the binary log and tables.
-
-Finally, the binary log position will be printed to `STDERR` and xtrabackup
-will exit returning 0 if all went OK.
-
-Note that the `STDERR` of xtrabackup is not written in any file. You will
-have to redirect it to a file, e.g., `xtrabackup OPTIONS 2> backupout.log`.
-
-It will also create the following files in the
-directory of the backup.
-
-During the `prepare` phase, Percona XtraBackup performs crash recovery against
-the copied data files, using the copied transaction log file. After this is
-done, the database is ready to restore and use.
-
-The backed-up MyISAM and InnoDB tables will be eventually consistent with
-each other, because after the prepare (recovery) process, InnoDB’s data is
-rolled forward to the point at which the backup completed, not rolled back to
-the point at which it started. This point in time matches where the `FLUSH
-TABLES WITH READ LOCK` was taken, so the MyISAM data and the prepared
-InnoDB data are in sync.
-
-The xtrabackup offers many features not mentioned in the preceding
-explanation. The functionality of each tool is explained in more
-detail further in this manual. In brief, though, the tools enable you
-to do operations such as streaming and incremental backups with
-various combinations of copying the data files, copying the log files,
-and applying the logs to the data.
+The application copies your InnoDB data files, resulting in internally inconsistent data, and performs crash 
```

Comment:
instead of application, we can say xtrabackup copies your.. looks better this imho